### DISCUSSION TITLE
Feature request for adding pngfix.exe to libpng port

### DISCUSSION THREAD

**Discussion by alinpahontu2912** [microsoft/vcpkg#42052] [Ideas]
Hey there, I am working on porting torchvision to Windows on Arm64. The official documentation requires libpng with the pngfix.exe tool installed in order to enable support for png files. While I can build the library and this tool myself, I think it would be of great help if the vcpkg port of libpng could provide this tool. Conda provides it out of the box, but unfortunately has no Windows Arm64 support, so I can't use it. This would make it easier for users to build the package locally and will greatly help in creating the official port of torchvision for Windows Arm64.

**Comment by Osyotr** [comment]
Please test https://github.com/microsoft/vcpkg/pull/42058

**Reply by alinpahontu2912** [reply]
Hey, thank you very much!
